### ISSUE TITLE
Add Chromium Edge launcher in karma

### ISSUE THREAD

**Issue by XhmikosR** [twbs/bootstrap#32044]
https://github.com/ChiragRupani/karma-chromiumedge-launcher

**Comment by XhmikosR** [comment]
I had a quick look and it seems https://github.com/litixsoft/karma-detect-browsers doesn't support Chromium Edge. Not sure if there's a way to bypass this and use the above package @Johann-S?

**Comment by Johann-S** [comment]
it seems https://github.com/karma-runner/karma-chrome-launcher handle Chromium, so I don't understand what's the issue here 🤔

**Comment by XhmikosR** [comment]
Chromium Edge != Chromium :)

But feel free to try; it didn't work for me when I tried it.

**Comment by Johann-S** [comment]
Oh ok I understand you mean Edge working with Chromium under the hood

**Comment by XhmikosR** [comment]
Yeah, I mean the new Edge Chromium which is preinstalled on Windows nowadays :)

**Comment by sykalerio** [comment]
"Hi there! My name is Syka Lerioti Elpida, and I'm new to the Bootstrap project. I came across issue #32044  and I'm interested in helping out with testing. I think this issue would be a great way for me to get started with contributing to the project. Could you provide me with some guidance on how to approach this issue? Thanks in advance for your help!"
